modules/aws/cassandra/cassandra.tf

```
@@ -96,6 +116,15 @@ resource "aws_ebs_volume" "cassandra_commitlog_volume" {
   availability_zone = local.location
   size              = local.cassandra.commitlog_remote_volume_size
   type              = local.cassandra.commitlog_remote_volume_type
+
+  tags = merge(
+    var.custom_tags,
+    {
+      Name      = "${local.network_name} Cassandra commitlog"
```

How about add ${count.index} to Name for distinguish instance ?

e.g.
```suggestion
      Name      = "${local.network_name} Cassandra commitlog - ${count.index + 1}"
```